app/src/main/res/layout/activity_forgot_password.xml

```diff
@@ -75,7 +75,7 @@
         android:layout_width="250dp"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/margin_medium"
-        android:background="@color/md_grey_300"
+        android:background="@drawable/rounded_button"
```

Comment:
I think i did that too, but forgot to attach images.
Can u please check? 
thanks.